site/docs/guidelines/infinite-scroll.mdx

```diff
@@ -1,20 +1,11 @@
 ---
-title: Infinite Scroll with a “View more” button
+title: Infinite Scroll
 navTitle: Infinite Scroll
 summaryParagraph: Infinite Scroll separates large bodies of content into separate sections. You can access each extra section via a "View more" button.
 tags: ["Infinite scrolling", "Continuous scroll", "Load more", "Async loading"]
 githubLabels: ["component:infinite-scroll"]
```

Comment:
Looks like guidelines should have these properties:

```md
// from guidelines/checkbox-radio-toggle-switch.mdx
title: "Checkbox, Radio, and Toggle Switch"
navTitle: "Checkbox, Radio, and Toggle Switch"
summaryParagraph:
tags:
needToKnow:
inComparingSection: true
```

So we should remove `githubLabels`.
`inComparingSection` appears to be optional though.

Reply:
Yeah looks like the `inComparingSection` attribute is used when you want the page to appear in the bottom 'Comparing Components' section:
<img width="353" alt="image" src="https://user-images.githubusercontent.com/1811583/157797800-c60afe96-76ff-4eb9-bcfe-b1822ca54164.png">

Reply:
I've removed the `githubLabels` frontmatter, and deleted the label from Github as well